client/src/app/components/stardust/Activity.tsx

```diff
@@ -0,0 +1,43 @@
+import { FunctionComponent } from 'react';
```

Comment:
Lets just use double-quotes `"` in imports everywhere.

Reply:
Done